src/mixins/LeadTime.js

```diff
@@ -0,0 +1,48 @@
+import { types } from 'mobx-state-tree';
+import debounce from  'lodash.debounce';
+
+const DEBOUNCE_INTERVAL = 500;
```

Comment:
Does this have the chance to miss the debounced setting of leadTime, where it doesn't get updated in time for draft save?

Reply:
I was also curious as to why we are debouncing it by 500ms?

Reply:
resolving after discussion with @juliosgarbi, this is meant to capture leadtime with a minimum interaction time, so repetitive typing is captured in a single window of time, and one-off typing gets considered as separate windows of 0.5 seconds in the leadtime calculation.

Reply:
Opening this back, as it's worth to add a comment about this var and it's purpose. Like "Minimal interaction time to count as well as debounce for multiple consequential interactions, so they can be counted as one long interaction window."

Reply:
resolving as there is no change requested

Reply:
em... that was an action requested, to add comment. we have a very few comments in a code and we should go toward more comments and cleaner and readable code.

Reply:
My apologies I misunderstood what you meant by comment. There are lots of comments in the file already would you settle for a more descriptive debounce variable name?